package.json

```
@@ -17,7 +17,7 @@
     "scripts:build:data": "node scripts/commands/build-data",
     "scripts:googleapis-generate-token": "node scripts/commands/googleapis-generate-token",
     "scripts:wizard": "node scripts/commands/wizard/index.js",
-    "sitemap": "rm public/sitemap.xml && vue-cli-service sitemap",
+    "sitemap": "rm public/sitemap.xml ; vue-cli-service sitemap",
```

nitpick, but if you want `rm` to succeed even if the file is missing, then use `rm -f`. If `rm` fails for some other reason, then you probably dont want to run `vue-cli-service sitemap`

```suggestion
    "sitemap": "rm -f public/sitemap.xml && vue-cli-service sitemap",
```